podcasts/PlayerTabsView.swift

```diff
@@ -21,6 +22,8 @@ enum PlayerTabs: Int {
             return L10n.chapters
         case .bookmarks:
             return L10n.bookmarks
+        case .transcripts:
+            return "Transcripts"
```

Comment:
I'm not localising the strings on this PR because these are temporary until we have final designs.

Reply:
I believe we can start doing so, unless is a very specific copy? "Transcripts" I'm sure we will have in the final version somehow.

Anyway, not a strong feeling.